src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -32,12 +32,30 @@ export class LocalTaskService implements TaskService {
     return of(null);
   }
 
+  setDone(id: string): Observable<void> {
```

Comment:
just a minor remark, but if you actually go for a toggle implementation here (`selectedTask.done = !selectedTask.done`), I'd tend more towards a name that reflects that. So either call it "toggleDone" or maybe make it even more explicit and add a boolean attribute to the method's signature.